3-platform/shopping-cart/app/src/main/java/com/_6core/platform/shopping/cart/app/validator/ValidationHandler.java

```diff
@@ -0,0 +1,37 @@
+package com._6core.platform.shopping.cart.app.validator;
+
+public abstract class ValidationHandler<T> implements Validator<T> {
+  private ValidationHandler<T> nextValidator;
+
+  public ValidationHandler<T> build(
+      final ValidationHandler<T> first, final ValidationHandler<T>... chain) {
+    ValidationHandler<T> head = first;
+    for (ValidationHandler<T> nextInChain : chain) {
+      head.nextValidator = nextInChain;
+      head = nextInChain;
+    }
+    return first;
+  }
```

Comment:
lets move logic of building chain to the `infra/configuration` using spring, e.g. @Autowired List<Validatior<ShoppingCartV01>>

Reply:
@sokimaaa  , I thought that 'order of validation' is important, so implemented a chain of responsibility.
With autowiring the list we can't control the order. Can we discuss it?